msteams-platform/concepts/device-capabilities/native-device-permissions.md

```diff
@@ -212,6 +212,17 @@ For example:
 
 * To prompt the user to share location on the map interface, Teams app asks permission when you call `getLocation()`:
 
+    # [TeamsJS v2](#tab/teamsjs-v2)
+
+    ```JavaScript
+     function getLocation() {
+        microsoftTeams.location.getLocation({ allowChooseLocation: true, showMap: true }).then((location) => { 
```

Comment:
in v2, we aren't generally using the microsoftTeams prefix anymore. Probably just

`location.getLocation(...)`